app/src/main/java/com/example/concordia_campus_guide/models/routes/TransportType.java

```diff
@@ -1,4 +1,4 @@
 package com.example.concordia_campus_guide.models.routes;
 
-public abstract class TransportType {
+public interface TransportType {
```

Comment:
i'm using "instanceof" in a class, don't remember where. Using interface will affect that part of the code no?

Reply:
@ferdousara-parvin Did it affect it when you ran the app?

Reply:
Wait, why is this interface not deleted? It has nothing in it ...

Reply:
Ok I got my answer after a quick search: https://en.wikipedia.org/wiki/Marker_interface_pattern

Reply:
"The java “instanceof” operator is used to test whether the object is an instance of the specified type (class or subclass or interface)."

Reply:
I did not delete it because it is implemented by multiple types of transport like Bus Walk etc...